pokedex/src/Components/App/App.jsx

```diff
@@ -0,0 +1,160 @@
+import React from "react";
+import Header from "../Header/Header";
+import Pokesprites from "../Pokesprites/Pokesprites";
+import Pokeinfo from "../Pokeinfo/Pokeinfo";
+import Aboutme from "../Aboutme/Aboutme";
+import { useState, useEffect } from "react";
+import "../App/App.css";
+import "../style.css";
+import Footer from "../Footer/Footer";
+import { Routes, Route } from "react-router-dom";
+import pokemonData from "../pokemonapi.json";
+import { useNavigate } from "react-router-dom";
+
+const App = () => {
+  const [loading, setLoading] = useState(true);
+  const [url, setUrl] = useState("https://pokeapi.co/api/v2/pokemon/");
+  const [activePokemon, setActivePokemon] = useState();
+  const [pokeInfo, setPokeInfo] = useState({});
+  const [searchTerm, setSearchTerm] = useState("");
+
+  console.log(searchTerm);
+
+  const [pokemonList, setPokemonList] = useState(pokemonData.results);
+  const filteredPokemonList = pokemonList.filter((pokemon) =>
+    pokemon.name.includes(searchTerm)
+  );
+
+  const pokeData = Object.values(pokeInfo).filter((pokemon) => {
+    const bool = pokemon.name.includes(searchTerm);
+    return bool;
+  });
+
+  const navigate = useNavigate();
+
+  const pokeFun = async () => {
+    setLoading(true);
+    const res = await fetch(url);
+    const data = await res.json();
+    fetchPokemonDetails(data.results);
+    setLoading(false);
+  };
```

Comment:
* One remark hasn't been corrected https://github.com/Kyle-Le-create/project-pokedex-frontend/pull/1/files#r1820082171  You need to `catch` possible errors at the very **end** of any server request. **Please, correct it everywhere in the code**


In this case you need to wrap the code into `try/catch`